guides/introduction/Embedded Schemas.md

```diff
@@ -219,4 +219,4 @@ user_changeset = User.changeset(%User{}, %{profile: %{online: true, visibility:
 #]
 ```
 
-In databases where `:map`s are stored as JSONB (like Postgres), Ecto constructs the appropriate jsonpath queries for you. More examples of embedded schema queries are documented in [`json_extract_path/2`](https://hexdocs.pm/ecto/Ecto.Query.API.html#json_extract_path/2).
\ No newline at end of file
+In databases where `:map`s are stored as JSONB (like Postgres), Ecto constructs the appropriate jsonpath queries for you. More examples of embedded schema queries are documented in [`json_extract_path/2`](https://hexdocs.pm/ecto/Ecto.Query.API.html#json_extract_path/2).
```

Comment:
I assume there was a missing newline here which my editor added. I can undo this change if you want to keep it to the typo only.